app/views/two_factor_authentication/otp_verification/show.html.erb

```diff
@@ -10,33 +10,34 @@
 
 <%= form_tag(:login_otp, method: :post, role: 'form', class: 'margin-top-4') do %>
   <%= render @presenter.reauthn_hidden_field_partial %>
-  <%= label_tag 'code',
-                t('simple_form.required.html') + t('forms.two_factor.code'),
-                class: 'block bold' %>
-  <div class="col-12 sm-col-5 margin-bottom-2 tablet:margin-bottom-0 sm-mr-20p inline-block">
-    <%= text_field_tag(:code, '',
-                       value: @presenter.code_value,
-                       required: true,
-                       autofocus: true,
-                       pattern: '[0-9]*',
-                       class: 'col-12 field monospace mfa',
-                       aria: { invalid: false, describedby: 'code-instructs' },
-                       maxlength: TwoFactorAuthenticatable::DIRECT_OTP_LENGTH,
-                       autocomplete: 'one-time-code', type: 'tel') %>
+  <div class="grid-row margin-bottom-5">
+    <div class="grid-col-12 tablet:grid-col-6">
+      <%= label_tag 'code',
+                    t('simple_form.required.html') + t('forms.two_factor.code'),
```

Comment:
I see this line is old code, just moved, but I'm surprised we don't have to `safe_join` these two, considering one has `.html` in it

Reply:
> I see this line is old code, just moved, but I'm surprised we don't have to `safe_join` these two, considering one has `.html` in it

🤔  Yeah, definitely feels suspect, though in thinking on it in some detail, I'm not able to come up with any real-world scenario where this might be problematic. I could add the `safe_join`, though... I'm not actually sure what the "required" HTML is doing for us anyways, since as best I can tell it's hidden and never exposed either visually or to assistive technology.

https://github.com/18F/identity-idp/blob/770e27bf68d02e9172cefd6546f3014e4660d542/config/locales/simple_form/en.yml#L8

Looks like it may have been originally [inherited from SimpleForm](https://github.com/heartcombo/simple_form/blob/2fa860e5f607da307570df75cf57b3684f82e23c/lib/simple_form/components/labels.rb#L8-L12), then at some point along the way we [decided we didn't want to show it anymore](https://github.com/18F/identity-idp/pull/403/files#diff-b9c876aa430c630577281ea66040f1b036bc50fc55277a594ac8709c4c85b004R10). Kinda seems like the easier option would just be to remove the HTML altogether, or at least not reference it here 🤷

Reply:
> maybe we just remove it then while we're here?

Yeah, I'll plan to at least remove it from the label on this screen. I'll plan a follow-up to remove it across the whole application.

Reply:
> > maybe we just remove it then while we're here?
> 
> Yeah, I'll remove it from the label on this screen. I'll plan a follow-up to remove it across the whole application.

Removed in e681cbc22.

Reply:
> > maybe we just remove it then while we're here?
> 
> Yeah, I'll remove it from the label on this screen. I'll plan a follow-up to remove it across the whole application.

Follow-up at 4855